libs/barista-components/button-group/src/button-group.ts

```diff
@@ -203,7 +203,11 @@ export class DtButtonGroupItem<T> extends _DtButtonGroupItem
     DtButtonGroupItemSelectionChange<T>
   >();
 
-  /** Whether the button-group item is selected. */
+  /**
+   * @deprecated
+   * @breaking-change Will be renamed from 'selected' to 'checked' with v 9.0.0.
+   * Whether the button-group item is selected.
+   */
```

Comment:
please provide the new input for `checked` and add the behaviour so that when we remove this one it already works. so that they can already use the new version.

Reply:
We shouldn't do that as it changes the public API. Let's hold off with updating this in a minor version and fix it in version 9.